http.c

```diff
@@ -2873,7 +3119,15 @@ evhttp_connection_connect_(struct evhttp_connection *evcon)
 		 */
 		evhttp_connection_cb_cleanup(evcon);
 		return (0);
-	}
+	}else {
+        if (evcon->connectcb != NULL){
```

Comment:
There are lots of style coding issues when spaces used instead of tabs:

```suggestion
		if (evcon->connectcb != NULL){
```